<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gauss</class>
 <widget class="QDialog" name="Gauss">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1113</width>
    <height>580</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1113</width>
    <height>580</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1113</width>
    <height>580</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Gauss_소거법_박무진_1724560</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(245, 245, 245);
</string>
  </property>
  <widget class="QPushButton" name="startBtn">
   <property name="geometry">
    <rect>
     <x>515</x>
     <y>240</y>
     <width>80</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(206, 255, 226);
font: 14pt &quot;Agency FB&quot;;
border-radius: 5px;
border: 1px solid black;
</string>
   </property>
   <property name="text">
    <string>연산</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="inputBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>500</width>
     <height>499</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>171</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>입력받은 행렬</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="outputBox">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>50</y>
     <width>500</width>
     <height>499</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>550</y>
     <width>411</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;맑은 고딕&quot;;
color: #383e56;
font-weight: 700;</string>
   </property>
   <property name="text">
    <string>수치해석 가우스_소거법 / Copyright 2021 moomin. All rights reserved</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>10</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>연산 결과</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
